.github/workflows/repo-sync.yml

```diff
@@ -0,0 +1,64 @@
+on:
+  schedule:
+    - cron:  '0 0 * * 1,4'
+  workflow_dispatch:
+    inputs:
+      branch_to_sync:
+        description: Branch to sync to the private repository
+        default: develop
+        required: true
```

Comment:
required is true but when this runs automatically the input won't be passed. will it still work?
or does it work because the default value is set to develop? meaning that required:true doesn't really do anything?

Reply:
explained as if there is not required, then the user can delete the workflow and run it with having the input being empty, whereas if this flag is included, it will prompt the user to write a value to the input